e2e/tests/pages/hearing.spec.js

```diff
@@ -0,0 +1,121 @@
+import { expect, test } from '@playwright/test';
+
+const API_URL = process.env.API_URL || 'https://kerrokantasi.api.dev.hel.ninja';
+
+const fetchHearing = async () => {
+  const res = await fetch(`${API_URL}/v1/hearing/`);
+  const json = await res.json();
+  const openHearings = json.results.filter((hearing) => !hearing.closed);
+  const hearingId = openHearings[0].id;
+  const hearing = await fetch(`${API_URL}/v1/hearing/${hearingId}/`);
+
+  return hearing.json();
+};
+
+const hearingContainsQuestions = (hearing) => Boolean(hearing.sections.some((s) => s.questions.length));
+
+test.describe('Hearing', () => {
+  let page;
+  let hearing;
+
+  test.beforeAll(async ({ browser }) => {
+    hearing = await fetchHearing();
+    page = await browser.newPage();
+    await page.goto(hearing.slug);
+    await page.getByTestId('cookie-consent-approve-required-button').click();
+  });
+
+  test.beforeEach(async () => {
+    await page.goto(hearing.slug);
+  });
+
+  test('should display Finnish title', async () => {
+    await expect(page.locator('main')).toContainText(hearing.title.fi);
+  });
+
+  test('should have comment link', async () => {
+    const link = page.locator('.internal-link:has-text("Kirjoita kommentti")');
+    await expect(link).toBeVisible();
+  });
+
+  test('should display comment count', async () => {
+    await expect(page.getByTestId('comment-summary')).toContainText(`Yhteensä ${hearing.n_comments} kommenttia`);
```

Comment:
Looks like there has been changes texts since last review 👀 :
https://dev.azure.com/City-of-Helsinki/kerrokantasi/_build/results?buildId=172816&view=logs&j=2fd77a48-9636-5e84-96b6-2d0003581ea6&t=b06928c6-3cc6-582d-2dda-b79f345b07f1&l=28

Reply:
Changed

`.toContainText("Yhteensä ${hearing.n_comments} kommenttia")`

to

`.toContainText("Yhteensä ${hearing.n_comments} kommentti")`


Now the test should pass in both cases: "1 kommentti" or "N kommenttia"